config.yml

```diff
@@ -0,0 +1,246 @@
+---
+CI_HELPER_VERSION: &CI_HELPER_VERSION 0.0.6
+ci_helper_test: &ci_helper_test ci-helper version -e
+docker_hub_namespace: &docker_hub_namespace ekino/docker-buildbox
+
+
+ansible:
+  "2.5.5-r0":
+    build_args:
+      CI_HELPER_VERSION: *CI_HELPER_VERSION
+      PYTHON_VERSION: 2.7.15-r1
+      VERSION: 2.5.5-r0
+    cmd_test:
+      - ansible --version
+      - ansible-playbook --version
+      - python --version
+
+arachni:
+  "1.5":
+    build_args:
+      ARACHNI_VERSION: 1.5.1
+      ARACHNI_WEB_UI_VERSION: 0.5.12
+      CI_HELPER_VERSION: *CI_HELPER_VERSION
+    cmd_test:
+      - arachni --version
+      - *ci_helper_test
+
+aws:
+  "1":
+    cmd_test:
+      - aws --version
+      - pip list | grep boto3
+      - pip list | grep pyyaml
+      - pip --version
+      - pipenv --version
+      - *ci_helper_test
+    build_args:
+      CI_HELPER_VERSION: *CI_HELPER_VERSION
+      PIP_VERSION: 18.1
+      PIPENV_VERSION: 2018.10.13
+      PY_YAML_VERSION: 3.13
+
+chrome:
+  "1":
+    build_args:
+      MODD_VERSION: 0.5
+    cmd_test:
+      - google-chrome --no-sandbox --version
+      - google-chrome-unstable --no-sandbox --version
+      - node --version
+      - npm --version
+      - yarn --version
+
+dind-aws:
+  "1":
+    build_args:
+      CI_HELPER_VERSION: *CI_HELPER_VERSION
+      DOCKER_COMPOSE_VERSION: 1.24.0
+      GLIBC_VERSION: 2.27-r0
+      PIP_VERSION: 18.1
+      PIPENV_VERSION: 2018.10.13
+    cmd_test:
+      - aws --version
+      - docker --version
+      - docker-compose --version
+      # - --privileged -d --name dind-aws alpine:latest
+      # - docker ps
+      - *ci_helper_test
+
+golang:
+  "1.11":
+    build_args:
+      CI_HELPER_VERSION: *CI_HELPER_VERSION
+      GLIDE_VERSION: v0.13.1
+      MODD_VERSION: 0.5
+    cmd_test:
+      - aws --version
+      - go version
+      - glide --version
+      - gin --version
+      - modd --version
+      - *ci_helper_test
+
+java_test: &java_test
+  - java -version
+  - mvn --version
+  - aws --version
+  - modd --version
+  - dot -V
+  - jq --version
+
+java:
+  "6":
+    build_args:
+      CI_HELPER_VERSION: *CI_HELPER_VERSION
```

Comment:
Sorry I just noticed now, but Java 6 does not install CI Helper, so you don't need to inject this variable here

Reply:
Terraform doesn't install it either by the way